src/main/java/org/folio/dew/batch/acquisitions/edifact/exceptions/EntitiesNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package org.folio.dew.batch.acquisitions.edifact.exceptions;
+
+public class EntitiesNotFoundException extends RuntimeException {
```

Comment:
can we re-use already existing NotFoundException ?

Reply:
Yes, removed this one and now use NotFoundException